recurrence/base.py

```diff
@@ -653,6 +651,16 @@ def __repr__(self):
     MO, TU, WE, TH, FR, SA, SU) = WEEKDAYS = list(map(lambda n: Weekday(n), range(7)))
 
 
+def to_utc(dt):
```

Comment:
This function has been moved to recurrence.base as it can be imported to other modules easily from this location.